app/services/publisher_transactions_getter.rb

```diff
@@ -87,7 +89,7 @@ def perform_offline
           "channel" => "#{channel.details.channel_identifier}",
           "amount" => "#{contribution_settlement_amount}",
           "settlement_currency" => publisher.default_currency,
-          "settlement_amount" => "56.81",
+          "settlement_amount" => OFFLINE_CONTRIBUTION_SETTLEMENT_AMOUNT,
```

Comment:
Did you mix these variables up? I think this is supposed to be the `OFFLINE_REFERRAL_SETTLEMENT_AMOUNT` and the other one is supposed to be the `18.18` value

Reply:
Good catch, I did switch them.  I don't think it makes a difference since the totals still add up, but I will revert just in case.